app/components/search/SearchResults/SearchResults.tsx

```diff
@@ -1,4 +1,4 @@
-import React, { useCallback } from "react";
+import React, { ReactNode, useCallback } from "react";
```

Comment:
**chore:** let's remove this too. notice we've let a lot of warnings through. i have some capacity and am happy to submit a PR to get rid of them